playbooks/deploy-external-ceph.yml

```diff
@@ -0,0 +1,253 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Pre-requisites for deploying external ceph
+  hosts: localhost
+  vars:
+    cifmw_ceph_dir: "{{ ansible_user_dir }}/ci-framework-data/manifests/ceph"
+    cifmw_ceph_node_num: "{{ cifmw_ceph_node_num | default(3) }}"
+  tasks:
+    - name: Prepare inventory for ceph nodes
```

Comment:
If I have `N` compute nodes how is the subset of `cifmw_ceph_node_num` ceph nodes identified?

The first `cifmw_ceph_node_num` of them or are they expected to have a certain name?

I think you might be relying on some convention, and that convention should at least be documented in  comment.